src/rtctools_heat_network/esdl/esdl_heat_model.py

```diff
@@ -1328,10 +1342,14 @@ def convert_gas_tank_storage(self, asset: Asset) -> Tuple[Type[GasTankStorage],
         """
         assert asset.asset_type in {"GasStorage"}
 
+        # hydrogen_specific_energy = 20.0 / 1.0e6  # kg/Wh
```

Comment:
Should this line be commented out? Else delete or add comment that it should not be deleted due to ....

Reply:
updated